.vale/fixtures/OpenShiftAsciiDoc/TrailingBackslash/testvalid.adoc

```diff
@@ -0,0 +1,39 @@
+[source,terminal]
+----
+$ oc get my lunch
+----
+
+[source,terminal]
+----
+# oc get my lunch
+----
+
+.Example
+[source,terminal]
+----
+$ az role assignment create --role "<privileged_role>" \// <1>
```

Comment:
Could you add a "normal" backslash without a callout to the testvalid file?

----
$ az z role assignment \
  create --role "<privileged_role>"
----

Reply:
This will trigger the rule. I'll see if I can refine the regex to cater for this.

Reply:
I ran into a similar issue when I tried to create a Vale rule that would flag lines in code that were missing backslashes. :-(  Never did find a good solution.

Reply:
Could handle it with a script rule for sure.